pymongo/encryption.py

```diff
@@ -256,6 +285,36 @@ def close(self):
             self.mongocryptd_client = None
 
 
+class RewrapManyDataKeyOpts(object):
+    def __init__(self, provider, master_key=None):
+        """Options given to a ``rewrap_many_data_key`` operation.
+
+        :Parameters:
+          - `provider`: The new KMS provider to use to encrypt the data keys,
+            or ``None`` to use the current KMS provider(s).
+          - ``master_key``: The master key fields corresponding to the new KMS
+            provider when ``provider`` is not ``None``.
+        """
+        self.provider = provider
+        self.master_key = master_key
+
+
+class RewrapManyDataKeyResult(object):
+    def __init__(self, raw_result=None):
+        """Result object returned by a ``rewrap_many_data_key`` operation.
+
+        :Parameters:
+          - `raw_result`: The result of the bulk write operation used to
+            update the key vault collection with rewrapped data keys.   If
+          - ``rewrap_many_data_key()`` does not find any matching keys to
+            rewrap, no bulk write operation will be executed and the ``bulk_write_result`` field will be unset.
+        """
+        if isinstance(raw_result, dict) and "bulk_write_result" in raw_result:
+            self.bulk_write_result = raw_result["bulk_write_result"]
+        else:
+            self.bulk_write_result = None
```

Comment:
We should expose bulk_write_result via a `@property` so that we can add a docstring for it.

Reply:
Done